src/stash/anchor.rs

```diff
@@ -145,15 +145,16 @@ impl Anchor {
         let tx = &mut psbt.global.unsigned_tx;
         let num_outs = tx.output.len() as u64;
 
+        // TODO: use enum instead of hardcoded value
+        let protocol_factor = 0;
+
```

Comment:
I do not think that setting the `protocol_factor` to zero is the best approach here. Besides, this is not compatible with what is done in the [`verify`](https://github.com/rgb-org/rgb-core/blob/39dfc511d167039d346408fab61ccff0d46e78b9/src/stash/anchor.rs#L284-L293) associated function of the `Anchor` structure.

So I propose that `protocol_factor` be initialize the same way that it is done there.

```suggestion
        let protocol_factor = (
            Uint256::from_be_bytes(*contract_id.as_slice()) % Uint256::from_u64(num_outs).unwrap()
        ).low_u64() as u32;
```